Library/Formula/openal-soft.rb

```diff
@@ -40,10 +31,7 @@ class OpenalSoft < Formula
   def install
     ENV.universal_binary if build.universal?
 
-    # Please don't reenable example building. See:
```

Comment:
Hmm, looks like my test was somehow broken, got build error this time. 
Rolled back this change.